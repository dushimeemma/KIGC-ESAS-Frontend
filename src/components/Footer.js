import React from 'react';

const Footer = () => {
  return (
    <div className='text-center back-color cursor mt-2'>
      &copy; {new Date().getFullYear()}. KIGC | ESAS
    </div>
  );
};

export default Footer;
