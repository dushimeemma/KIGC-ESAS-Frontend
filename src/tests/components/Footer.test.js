import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../components/Footer';

describe('<Footer/>', () => {
  it('Should render footer without error', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
