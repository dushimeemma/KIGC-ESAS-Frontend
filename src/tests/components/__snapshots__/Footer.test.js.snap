// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> Should render footer without error 1`] = `
<div
  className="text-center back-color cursor mt-2"
>
  Â© 
  2020
  . KIGC | ESAS
</div>
`;
